Sources/MapboxNavigation/NavigationMapView+VanishingRouteLine.swift

```diff
@@ -332,7 +335,7 @@ extension NavigationMapView {
     /**
      Given a congestion level, return its associated color.
      */
-    func congestionColor(for congestionLevel: String?, isMain: Bool) -> UIColor {
+    func congestionColor(for congestionLevel: String? = nil, isMain: Bool) -> UIColor {
```

Comment:
Change the call site to pass in `nil` instead of providing a default value here. According to the Swift API design guidelines, parameters that have default argument values go at the end of the parameter list, but that change doesn’t seem to be necessary here.

Reply:
Thanks for the suggestion and the reference to the Swift API design guidelines, I've removed the default value here.